### ISSUE TITLE
The pipeline (sarek 2.7.1) stopped working once I updated to Nextflow version 22.04.0

### ISSUE THREAD

**Issue by SzilviaKiriakov** [nf-core/sarek#530]
The error message I received once I updated to Nextflow version 22.04.0:

> No such variable: ch_software_versions_yaml
>  -- Check script '.nextflow/assets/nf-core/sarek/main.nf' at line: 450 or see '.nextflow.log' file for more details

Seemed like `ch_software_versions_yaml` is not defined correctly. I copied `software_versions_mqc.yaml` into sarek/assets and added the following line based on nf-core/rnaseq pipeline code (which runs fine with the new Nextflow version) and commented out line 450:

>ch_software_versions_yaml = file("$projectDir/assets/software_versions_mqc.yaml", checkIfExists: true)

After this I reran the test, and received this error message:

>No such variable: bwa_built
> -- Check script 'sarek/main.nf' at line: 484 or see '.nextflow.log' file for more details

`bwa_built` variable is also defined by the command `file   into bwa_built`, similarly to `ch_software_versions_yaml`, maybe this type of variable definition is not supported in Nextflow 22.04.0 anymore.

If I run `NXF_VER=21.10.4 nextflow run nf-core/sarek -profile test,docker` the test pipeline runs fine.

**Comment by FriederikeHanssen** [comment]
Hi! Yes this is expected since the newest update uses DSL2 as default. Some more explanation here: https://nfcore.slack.com/archives/CE6P95170/p1649672480654609

TL;DR: try exporting this or prepending it to your run command: `export NXF_DEFAULT_DSL=1`

**Comment by SzilviaKiriakov** [comment]
Unfortunately I cannot open the Slack link, but `export NXF_DEFAULT_DSL=1` makes sense. Thank you for your help!

**Comment by FriederikeHanssen** [comment]
Maybe you are not part of the slack space. If you want to join: https://nf-co.re/join . 
Great you could work it out
